packages/composables/src/hooks/useCart/index.ts

```diff
@@ -2,23 +2,32 @@ import { ref, Ref, computed } from "@vue/composition-api";
 import {
   getCart,
   addProductToCart,
+  addPromotionCode,
   removeCartItem,
   changeCartItemQuantity,
 } from "@shopware-pwa/shopware-6-client";
 import { ClientApiError } from "@shopware-pwa/commons/interfaces/errors/ApiError";
 import { Cart } from "@shopware-pwa/commons/interfaces/models/checkout/cart/Cart";
 import { Product } from "@shopware-pwa/commons/interfaces/models/content/product/Product";
 import { LineItem } from "@shopware-pwa/commons/interfaces/models/checkout/cart/line-item/LineItem";
-import { getApplicationContext } from "@shopware-pwa/composables";
+import {
+  getApplicationContext,
+  useNotifications,
+} from "@shopware-pwa/composables";
 import { ApplicationVueContext } from "../../appContext";
 
+const TYPE_PROMOTION = "promotion";
```

Comment:
this could be a type in `commons` package
example
```
export type CartItemType = "promotion" | "product"
```

Reply:
You're right and there even is a CartItemType interface already with the needed types... I'll use that one.